src/com/geekhub/hw2/vehicle/Detail.java

```diff
@@ -0,0 +1,10 @@
+package com.geekhub.hw2.vehicle;
+
+/**
+ * Created by COMRADE on 17.10.2015.
+ */
+public abstract class Detail {
```

Comment:
Deatil is very abstract term. Better use more concrete term, forexample RunMechanizm.